x-pack/plugins/license_management/kibana.json

```diff
@@ -1,5 +1,9 @@
 {
   "id": "licenseManagement",
+  "owner": {
```

Comment:
It is the "Stack management" team who owns the license management app 😊

Reply:
@sebelga I removed the core team as the owner of `licenseManagement`.